### PR TITLE
Helm chart: configurable liveness and readiness probes

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

**Review by dimas-b — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ as necessary. Empty sections will not end in the release notes.
 
 ### New Features
 
+- Helm chart: liveness and readiness probes are now configurable via the `livenessProbe` and 
+  `readinessProbe` Helm values.
```
Comment (snazy) [inline CHANGELOG.md:18]: No `{{` - so :+1:

File: helm/nessie/values.yaml [hunk 2]
```diff
@@ -716,3 +716,31 @@ affinity: {}
 #                operator: In
 #                values:
 #                  - nessie
+
+# -- Configures the liveness probe for nessie pods.
+livenessProbe:
+  # -- Number of seconds after the container has started before liveness probes are initiated. Minimum value is 0.
+  initialDelaySeconds: 5
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 30
```
Comment (snazy) [inline helm/nessie/values.yaml:725]:
```suggestion
  periodSeconds: 10
```

File: helm/nessie/values.yaml [hunk 3]
```diff
@@ -716,3 +716,31 @@ affinity: {}
...
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+  # -- Optional duration in seconds the pod needs to terminate gracefully upon probe failure. Minimum value is 1.
+  terminationGracePeriodSeconds: 30
+
+# -- Configures the readiness probe for nessie pods.
+readinessProbe:
+  # -- Number of seconds after the container has started before readiness probes are initiated. Minimum value is 0.
+  initialDelaySeconds: 10
```
Comment (snazy) [inline helm/nessie/values.yaml:738]:
```suggestion
  initialDelaySeconds: 5
```

File: helm/nessie/values.yaml [hunk 4]
```diff
@@ -716,3 +716,31 @@ affinity: {}
...
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 45
```
Comment (snazy) [inline helm/nessie/values.yaml:740]:
```suggestion
  periodSeconds: 10
```

**Review by snazy — COMMENTED** [review]
Just suggestion to lower the initial and delay values

**Comment by adutra** [comment]
> Just suggestion to lower the initial and delay values

There is a complex equation to solve here: if the first probe kicks in earlier, that's generally a good thing... unless it fails, in which case, we'd have to pay the price of waiting `periodSeconds` until the next probe.

So if there is little chance for the probe to succeed at 5 seconds, maybe it's better to let it kick in at, say, 10 seconds.

But with all that said, I'm OK to make 5 seconds the default, especially if the period is 10 seconds instead of 30.

**Review by dimas-b — APPROVED** [review]

**Review by snazy — APPROVED** [review]

**Pull request merged**